iree/turbine/kernel/wave/templates/extend_attention.py

```diff
@@ -69,6 +70,8 @@ def get_extend_attention_kernel(
     N_WAVES = 1
     LOG2E = 1.44269504089
     logit_cap *= LOG2E
+    dk_sqrt = math.sqrt(1.0 / shape.head_size)
+    layer_scaling = layer_scaling or dk_sqrt * LOG2E
```

Comment:
would this be same as `(layer_scaling or dk_sqrt) * LOG2E?

Reply:
good catch, that's a bug. will fix